pytest_fixtures/core/contenthosts.py

```diff
@@ -21,7 +21,7 @@ def host_conf(request):
     _rhelver = f"{distro}{params.get('rhel_version', settings.content_host.default_rhel_version)}"
     # check to see if no-containers is passed as an argument to pytest
     deploy_kwargs = {}
-    if not request.config.getoption('no-containers', None) and not params.get('no_containers'):
+    if not request.config.getoption('no_containers') and not request.node.get_closest_marker('no_containers'):
```

Comment:
The params check needs to stay since other fixtures are using it to control whether or not to use containers. Of course, this line will need to be run through black.
```suggestion
    if not request.config.getoption('no_containers') and not params.get('no_containers') and not request.node.get_closest_marker('no_containers'):
```

Reply:
Ohh! Make sense! Reinitiating the removed check !

Reply:
Fixed !